__tests__/app.test.js

```diff
@@ -102,4 +102,52 @@ describe("GET /api/articles", () => {
     });
   });
 
+  describe("GET /api/articles/:article_id/comments", () => {
+    it("responds with an array of comments for the given article_id with the required properties", () => {
+      return request(app)
+        .get("/api/articles/1/comments")
+        .expect(200)
+        .then((res) => {
+          const comments = res.body;
+          comments.forEach((comment) => {
+            expect(comment).toMatchObject({
+              comment_id: expect.any(Number),
+              votes: expect.any(Number),
+              created_at: expect.any(String),
+              author: expect.any(String),
+              body: expect.any(String),
+              article_id: expect.any(Number),
+            });
+          });
+        });
+    });
+    it("serves the most recent comments first", () => {
+      return request(app)
+        .get("/api/articles/1/comments")
+        .expect(200)
+        .then((res) => {
+          const comments = res.body;
+          expect(comments.length).toBeGreaterThan(0);
+          const timestamps = comments.map((comment) =>
```

Comment:
You create these timestamps, but don't appear to be doing anything with them. My advice would be to use jest-sorted here to check if the created_at value is in a specific order